Makefile

```diff
@@ -71,10 +69,8 @@ kube-rc:
 	kubectl create -f ${RC}
 
 kube-clean:
-	kubectl delete rc deis-builder
-
-test:
-	@echo "Implement functional tests in _tests directory"
```

Comment:
should I rename the [above `test` target](https://github.com/deis/builder/pull/15#discussion-diff-47140079R48) then?